app/src/main/res/values-es/strings.xml

```diff
@@ -1,17 +1,44 @@
 <resources>
+    <string name="app_name">SDK de mapas y navegación para Android.</string>
```

Comment:
Are we OK adding `.` at the end of the string? The root `strings.xml` doesn't include it. The same thing happens with `description_navigation_route_ui`, `title_navigation_launcher`, `description_end_navigation`, `title_dual_navigation_map`, `description_dual_navigation_map`, `title_waypoint_navigation`, `description_waypoint_navigation`, `title_embedded_navigation`, `description_embedded_navigation`, `title_fragment_navigation`, `description_fragment_navigation`, `settings`, `simulate_route`, `route_profile`, `error_route_not_available`, `error_select_longer_route` and `explanation_long_press_waypoint`.